dialogue-core/src/test/resources/tracing/ChannelsTest/traces_on_retries.log

```diff
@@ -1,22 +1,26 @@
-{"traceId":"3af31cbecda2b46c","parentSpanId":"697e878d5bfc5628","spanId":"28f98da9021fd19b","type":"LOCAL","operation":"Dialogue-request-attempt","startTimeMicroSeconds":1582562924532338,"durationNanoSeconds":408967,"metadata":{}}
-{"traceId":"3af31cbecda2b46c","parentSpanId":"b430069d0282a2c9","spanId":"bfd65f0cca09c11b","type":"LOCAL","operation":"Dialogue-http-request initial","startTimeMicroSeconds":1582562924533529,"durationNanoSeconds":208896,"metadata":{}}
-{"traceId":"3af31cbecda2b46c","parentSpanId":"d7faec679d55f2d4","spanId":"b430069d0282a2c9","type":"LOCAL","operation":"Dialogue-http-request","startTimeMicroSeconds":1582562924533520,"durationNanoSeconds":229953,"metadata":{}}
-{"traceId":"3af31cbecda2b46c","parentSpanId":"d5319f25e6638359","spanId":"d7faec679d55f2d4","type":"LOCAL","operation":"Dialogue-request-attempt initial","startTimeMicroSeconds":1582562924533488,"durationNanoSeconds":334775,"metadata":{}}
-{"traceId":"3af31cbecda2b46c","parentSpanId":"697e878d5bfc5628","spanId":"d5319f25e6638359","type":"LOCAL","operation":"Dialogue-request-attempt","startTimeMicroSeconds":1582562924533473,"durationNanoSeconds":355345,"metadata":{}}
-{"traceId":"3af31cbecda2b46c","parentSpanId":"4a1d1c1893452979","spanId":"87b2d147957639a2","type":"LOCAL","operation":"Dialogue-http-request initial","startTimeMicroSeconds":1582562924534981,"durationNanoSeconds":333038,"metadata":{}}
-{"traceId":"3af31cbecda2b46c","parentSpanId":"749bb99b3951da3f","spanId":"4a1d1c1893452979","type":"LOCAL","operation":"Dialogue-http-request","startTimeMicroSeconds":1582562924534969,"durationNanoSeconds":363180,"metadata":{}}
-{"traceId":"3af31cbecda2b46c","parentSpanId":"28686cc7c2695fb2","spanId":"749bb99b3951da3f","type":"LOCAL","operation":"Dialogue-request-attempt initial","startTimeMicroSeconds":1582562924534927,"durationNanoSeconds":544058,"metadata":{}}
-{"traceId":"3af31cbecda2b46c","parentSpanId":"697e878d5bfc5628","spanId":"28686cc7c2695fb2","type":"LOCAL","operation":"Dialogue-request-attempt","startTimeMicroSeconds":1582562924534905,"durationNanoSeconds":577068,"metadata":{}}
-{"traceId":"3af31cbecda2b46c","parentSpanId":"efaf41a21cc7c6a9","spanId":"697e878d5bfc5628","type":"LOCAL","operation":"Dialogue-request initial","startTimeMicroSeconds":1582562924525312,"durationNanoSeconds":13461016,"metadata":{}}
-{"traceId":"3af31cbecda2b46c","parentSpanId":null,"spanId":"efaf41a21cc7c6a9","type":"LOCAL","operation":"Dialogue-request","startTimeMicroSeconds":1582562924525264,"durationNanoSeconds":13530208,"metadata":{}}
+{"traceId":"09b26a1be498af5c","parentSpanId":"49c88b0d4a7b609a","spanId":"802d8f9c0d0333a7","type":"LOCAL","operation":"Dialogue-http-request initial","startTimeMicroSeconds":1582681721129000,"durationNanoSeconds":7754255,"metadata":{}}
```

Comment:
https://5436-164943450-gh.circle-artifacts.com/0/~/artifacts/ChannelsTest/traces_on_retries/actual.html

Reply:
There is a lot of empty space in these - can we get a span containing the backoff duration in there somehow? (Could be as a FLUP)

Reply:
Ya, I'd prefer to do this in a follow-up